pkg/asset/quota/openstack/openstack.go

```diff
@@ -14,9 +14,10 @@ import (
 // These numbers should reflect what is documented here:
 // https://github.com/openshift/installer/tree/master/docs/user/openstack
 // https://github.com/openshift/installer/blob/master/docs/user/openstack/kuryr.md
-// Number or ports here don't include the constraints needed for each machine, which are calculated later
-var networkConstraint = buildNetworkConstraint(15, 1, 1, 1, 3, 60)
-var networkConstraintWithKuryr = buildNetworkConstraint(1500, 1, 250, 250, 250, 1000)
+// Number of ports, routers, subnets and routers here don't include the constraints needed
+// for each machine, which are calculated later
+var minNetworkConstraint = buildNetworkConstraint(15, 0, 0, 0, 3, 60)
+var minNetworkConstraintWithKuryr = buildNetworkConstraint(1500, 1, 250, 250, 250, 1000)
```

Comment:
This should probably have been set to:
```suggestion
var minNetworkConstraintWithKuryr = buildNetworkConstraint(1500, 0, 249, 245, 250, 1000)
```